### DISCUSSION TITLE
Debia 12 fresh install PHP 8.1 Error on POWERADMIN installer

### DISCUSSION THREAD

**Discussion by xxMAKCxx** [poweradmin/poweradmin#586] [Q&A]
Fatal error: Uncaught Error: Class "Twig\AbstractTwigCallable" not found in /var/www/html/vendor/twig/twig/src/TwigFilter.php:24 Stack trace: #0 /var/www/html/vendor/composer/ClassLoader.php(576): include() #1 /var/www/html/vendor/composer/ClassLoader.php(427): Composer\Autoload\{closure}() #2 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(199): Composer\Autoload\ClassLoader->loadClass() #3 /var/www/html/vendor/twig/twig/src/ExtensionSet.php(437): Twig\Extension\CoreExtension->getFilters() #4 /var/www/html/vendor/twig/twig/src/ExtensionSet.php(427): Twig\ExtensionSet->initExtension() #5 /var/www/html/vendor/twig/twig/src/ExtensionSet.php(395): Twig\ExtensionSet->initExtensions() #6 /var/www/html/vendor/twig/twig/src/Environment.php(856): Twig\ExtensionSet->getUnaryOperators() #7 /var/www/html/vendor/twig/twig/src/Lexer.php(540): Twig\Environment->getUnaryOperators() #8 /var/www/html/vendor/twig/twig/src/Lexer.php(125): Twig\Lexer->getOperatorRegex() #9 /var/www/html/vendor/twig/twig/src/Lexer.php(175): Twig\Lexer->initialize() #10 /var/www/html/vendor/twig/twig/src/Environment.php(504): Twig\Lexer->tokenize() #11 /var/www/html/vendor/twig/twig/src/Environment.php(551): Twig\Environment->tokenize() #12 /var/www/html/vendor/twig/twig/src/Environment.php(381): Twig\Environment->compileSource() #13 /var/www/html/vendor/twig/twig/src/Environment.php(343): Twig\Environment->loadTemplate() #14 /var/www/html/vendor/twig/twig/src/Environment.php(306): Twig\Environment->load() #15 /var/www/html/install/helpers/InstallationHelper.php(69): Twig\Environment->render() #16 /var/www/html/install/helpers/InstallationHelper.php(77): PoweradminInstall\InstallationHelper->renderTemplate() #17 /var/www/html/install/helpers/Installer.php(67): PoweradminInstall\InstallationHelper->step1ChooseLanguage() #18 /var/www/html/install/helpers/Installer.php(60): PoweradminInstall\Installer->handleStep() #19 /var/www/html/install/index.php(34): PoweradminInstall\Installer->initialize() #20 {main} thrown in /var/www/html/vendor/twig/twig/src/TwigFilter.php on line 24

**Comment by edmondas** [comment]
@xxMAKCxx What version is causing this error? Latest 3.8.1 or master branch?

**Comment by xxMAKCxx** [comment]
It was master. Download 3.8.1 and it's ok.

**Reply by edmondas** [reply]
I'm currently doing quite large changes in the installer, so the master might have temporary issues if those aren't caught by the minimal checks that are done after each commit, but anyway thanks for the report, I'll try to check if the issue is still there.

**Comment by edmondas** [comment]
In any case, a tip to fix such a problem is to run composer to reinstall all non-development dependencies:
```
composer install --no-dev
```
